examples/ray_tracing_triangle_meshes/source/ray_tracing_triangle_meshes.cpp

```diff
@@ -211,13 +211,16 @@ class ray_tracing_triangle_meshes_app : public gvk::invokee
 
 		mPipeline.enable_shared_ownership(); // Make it usable with the updater
 		mUpdater.emplace();
+		
+		// As we are updating our image views if swap chain is resized, it is advised to also cleanup our descriptor cache
+		std::function<void(void)> descriptorCleanup = [this]() {this->mDescriptorCache.cleanup(); };
+		mUpdater->on(gvk::swapchain_resized_event(gvk::context().main_window())).call(descriptorCleanup);
```

Comment:
I don't like that it is called `call`. Lambdas/functions are usually described to be _invoked_ not _called_. There's also `std::invoke` and I think we should stick to that terminology.